doc/classes/Engine.xml

```diff
@@ -114,7 +114,7 @@
 				[code]status[/code]   - Holds the status (e.g. "beta", "rc1", "rc2", ... "stable") as a String
 				[code]build[/code]    - Holds the build name (e.g. "custom_build") as a String
 				[code]hash[/code]     - Holds the full Git commit hash as a String
-				[code]year[/code]     - Holds the year the version was released in as an int
```

Comment:
That's wrong, it's "the year the version was released in", i.e. "the version was released in year YYYY".